spec/lib/nfg_ui/components/elements/slat_item_spec.rb

```diff
@@ -111,6 +188,53 @@
         end
       end
     end
+
+    describe 'complex tooltip implementations' do
```

Comment:
@farhajawed this spec here (slat_item_spec) may be useful in reviewing the complexity on the slat item tooltip. This spec goes through and outlines the scenarios as well as expectations I was referring to in the slat_item.rb file above.